MapboxVision/src/test/java/com/mapbox/vision/dsl/VisionTestDsl.kt

```diff
@@ -11,13 +14,15 @@ object TestCase {
     private const val GIVEN_PREFIX_NAME = "Given"
     private const val WHEN_PREFIX_NAME = "When"
     private const val THEN_PREFIX_NAME = "Then"
+    private const val VERIFY_PREFIX_NAME = "Verify"
 
     internal enum class WhenPrefix(val prefixName: String) {
         WHEN(WHEN_PREFIX_NAME)
     }
 
     internal enum class ThenPrefix(val prefixName: String) {
-        THEN(THEN_PREFIX_NAME)
+        THEN(THEN_PREFIX_NAME),
```

Comment:
Could you please advice where are this funs (`THEN` etc.) coming from? Cant find them in our or mockk sources...